packages/lab/src/dropdown/useDropdownBase.ts

```diff
@@ -85,10 +77,12 @@ export const useDropdownBase = ({
   }, [onOpenChange, setIsOpen]);
 
   const handleTriggerToggle = useCallback(
-    (e) => {
+    (e: MouseEvent) => {
       // Do not trigger menu open for 'Enter' and 'SPACE' key as they're handled in `handleKeyDown`
       if (
-        ["Enter", " "].indexOf((e as KeyboardEvent<HTMLDivElement>).key) === -1
+        ["Enter", " "].indexOf(
+          (e as unknown as KeyboardEvent<HTMLDivElement>).key
```

Comment:
Confused as to why we're checking for `key` on a mouseEvent

Reply:
I think this is because onClick is called by space and enter by default on browsers.

Reply:
Ah , interesting!